### DISCUSSION TITLE
Will there be RC xApp available?

### DISCUSSION THREAD

**Discussion by pengnist** [srsran/oran-sc-ric#10] [General]
Hi community, we are looking to use the KPM xApp with an RC xApp to perform some RAN control and am wondering if there is any RC xApp example available in python? 

Thanks

**Comment by mas-oliveira** [comment]
I think that srsRAN is working on an implementation of a RC module, I think they're working on something related to PRB management to make a demo how to use the RC interface

**Comment by pgawlowicz** [comment]
Indeed, we will soon publish an example.
